tests/e2e/package.json

```diff
@@ -17,7 +17,7 @@
     "test": "yarn test:chrome",
     "lint": "eslint . --ext .ts,.js,.tsx,.jsx",
     "test:desktop:ci": "ts-node ./desktop.runner.ts",
-    "test:desktop": "testcafe electron tests/ --browser-init-timeout 180000 -e -r html:./report/desktop-report.html,spec -q attemptLimit=3,successThreshold=1"
+    "test:desktop": "testcafe electron tests/ --browser-init-timeout 180000 -e -r html:./report/desktop-report.html,spec -q attemptLimit=3,successThreshold=1 -s"
```

Comment:
I think better use with `-S, --screenshots-on-fails                     take a screenshot whenever a test fails`
BTW, let's say we have a screenshot, does it show on circleci?